src/edu/washington/escience/myria/operator/agg/UserDefinedAggregatorFactory.java

```diff
@@ -157,16 +156,35 @@ private ScriptEvalInterface getEvalScript(
           .append(expr.getJavaExpression(param))
           .append(";\n");
 
-      // result.putType(I, valI);
-      output
-          .append(Expression.RESULT)
-          .append(".put")
-          .append(type.toJavaObjectType().getSimpleName())
-          .append("(")
-          .append(varCount)
-          .append(", val")
-          .append(varCount)
-          .append(");\n");
+      if (param.getStateSchema() == null) {
+        // state.putType(I, valI);
```

Comment:
It's to show what expression is being constructed. I simply followed the format of the previous comment.